### DISCUSSION TITLE
分享点图

### DISCUSSION THREAD

**Discussion by ffxpmu** [share121/inter-knot#2627] [其他]
![1723729396768](https://github.com/user-attachments/assets/283fe043-922f-497d-911c-040d476e19cb)
![1723813358545](https://github.com/user-attachments/assets/68fbc71f-77cd-421e-82ae-67e65eaeec14)
![ホタル_121493874](https://github.com/user-attachments/assets/a71f0fbc-cc22-4b54-9041-d8f880e2afd7)

**Comment by woll-lood** [comment]
-... ..- -- -... .-.. --- ...- . ... .. - -.-.--
